Makefile

```diff
@@ -99,6 +107,16 @@ FAILPOINT := tools/bin/failpoint-ctl
 FAILPOINT_ENABLE  := $$(echo $(FAILPOINT_DIR) | xargs $(FAILPOINT) enable >/dev/null)
 FAILPOINT_DISABLE := $$(echo $(FAILPOINT_DIR) | xargs $(FAILPOINT) disable >/dev/null)
 
+# go test -p parameter for unit tests
+P=3
+
+# The following packages are used in unit tests.
+# Add new packages here if you want to include them in unit tests.
+UT_PACKAGES_DISPATCHER := ./pkg/sink/mysql/... ./pkg/sink/util/... ./downstreamadapter/sink/... ./downstreamadapter/dispatcher/... ./downstreamadapter/worker/... ./pkg/sink/codec/open/... ./pkg/sink/codec/canal/...
```

Comment:
Could you move these paths to a separate file? Keeping them in this Makefile would make it too verbose.

Reply:
No, these items must be defined as Makefile variables. Moreover, if you need to add all the UTs for a new module, you must define a new variable and append it to the definition of unit_test_in_verify_ci. This way, everything must be kept in the same file.

Reply:
Once the UTs are fully organized, these codes will no longer be needed, and it won’t be necessary to define the path for each individual UT. All UTs can be run at once directly. These codes are only for temporary use until all the UTs are included.